label_studio_ml/examples/segment_anything_2_image/model.py

```diff
@@ -9,14 +10,17 @@
 from label_studio_sdk.converter import brush
 from label_studio_sdk._extensions.label_studio_tools.core.utils.io import get_local_path
 from PIL import Image
+
+ROOT_DIR = os.getcwd()
+sys.path.insert(0, ROOT_DIR)
 from sam2.build_sam import build_sam2
 from sam2.sam2_image_predictor import SAM2ImagePredictor
 
 
 DEVICE = os.getenv('DEVICE', 'cuda')
-SEGMENT_ANYTHING_2_REPO_PATH = os.getenv('SEGMENT_ANYTHING_2_REPO_PATH', 'segment-anything-2')
-MODEL_CONFIG = os.getenv('MODEL_CONFIG', 'sam2_hiera_l.yaml')
-MODEL_CHECKPOINT = os.getenv('MODEL_CHECKPOINT', 'sam2_hiera_large.pt')
+SSEGMENT_ANYTHING_2_REPO_PATH = os.getenv('SEGMENT_ANYTHING_2_REPO_PATH', 'sam2')
```

Comment:
is it a typo?
do we use this variable anywhere now?

Reply:
typo, and it's not used anywhere so I deleted it.